templates/Database/MsSql/split-connection-string.yaml

```diff
@@ -0,0 +1,33 @@
+parameters:
+  - name: 'connectionString'
+    type: string
+  - name: 'taskName'
+    type: string
+
+steps:
+  - powershell: |
+        $connectionString = "${{ parameters.connectionString }}"
+
+        $connectionValues = $connectionString -replace ';',"`r`n" |ConvertFrom-StringData
```

Comment:
prob a silly question, but why did you replace ; with ' instead of just splitting on ; ?

Reply:
because the code I copy and pasted had it in there haha